docs/user/integrate-otlp-backend/migration-to-otlp-logs.md

```diff
@@ -15,8 +15,50 @@ When you want to migrate to the `otlp` output, create a new LogPipeline. To prev
 
 You can't modify an existing LogPipeline to change its output type. You must create a new resource.
 
+See the following mapping of deprecated fields to their new OTLP-based counterparts:
+
+| Deprecated Field | Migration Action |
+|:--:|:--:|
+| spec.output.http or spec.output.custom | Required. Replace with spec.output.otlp. |
+| spec.filters                           | Rewrite the custom Fluent Bit filters using transform or filter expressions.|
+| spec.variables and spec.files          | These fields were used by custom filters. This functionality is now handled by transform or filter expressions.|
+| spec.input.application.dropLabels      | This field is no longer used. Configure label enrichment in the central Telemetry resource.|
+| spec.input.application.keepAnnotations | This functionality is not supported with the OTLP output and cannot be migrated.|
+
 ## Procedure
 
+1. Identify deprecated fields in your LogPipeline.
+
+   If your LogPipeline uses the `http` or `custom` output, you must migrate it to the OTLP stack and replace the deprecated fields:
+
+    ```yaml
+    apiVersion: telemetry.kyma-project.io/v1alpha1
+    kind: LogPipeline
+    metadata:
+      name: my-http-pipeline
+    spec:
+      input:
+        application:             # OTLP supports the application input, but you must replace the dropLabels and keepAnnotation flags
+          dropLabels: true       # Configure label enrichment centrally
+          keepAnnotations: true  # no longer supported
+      filters:
+        custom: |                # Replace with a transform or filter expression
+          ...
+      variables:                 # Used for custom filters, replace with a transform or filter expression
+        - name: myVar
+          value: myValue
+      files:                     # Used for custom filters, replace with transform or filter expressions
+        - name: myFile
+          value: |
+            ...
+      output:
+        http:                    # Switch to OTLP
+          endpoint:
+            value: "my-backend:4317"
+        custom: |                # Switch to OTLP
+          ...
+    ```
+
 1. Create a new LogPipeline that uses the `otlp` output.
 
     Pay special attention to the following settings (for details, see [Integrate With Your OTLP Backend](migration-to-otlp-logs.md)):
```

Comment:
The link seems strange, it links to the guide that I'm reading.

Reply:
good catch :)

Reply:
fixed